src/homework2/task7.py

```diff
@@ -18,13 +18,15 @@
    сумму чисел, расположенных между 1 и N включительно.
 """
 
+
 a = int(input())
 c = 0
-if (a > 0) and (a <= 10 ** 4):
-    s = (a + 1) * a // 2
-elif (a <= 0) and (a > ((-1) * 10 ** 4)):
-    s = (a + 1) * (abs(a) + 2 * (a <= 0)) // 2
-sign = -1 if a <= 0 else 1
+if 0 < a <= 10 ** 4:
```

Comment:
we shouldn't check var values if we have these limits in task description
(see the previous issue)